src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double getArea();
```

Comment:
if you've named interface like Calculator - let him calculate)
```suggestion
    double calculateArea();
```